source/guides/basics/mechanisms.md

```diff
@@ -1,49 +1,150 @@
-Mechanisms And Properties (PLACEHOLDER ONLY)
+Mechanisms And Properties
 -------------------------
 
 ```eval_rst
 .. contents:: Table of Contents
     :local:
 ```
 
-### Placeholder
+### What's a Mechanism?
 
-Until this page is written, you can view the [old tutorial video here](https://one.denizenscript.com/denizen/vids/Properties%20and%20Mechanisms).
+If you've made it this far, you've learned about [definitions](/guides/basics/definitions) and [flags](/guides/basics/flags), the two most common ways to read and write stored data for use in your scripts. If you're reading this guide for the first time, though, you're probably here to *change* data, not just read and write it. **Mechanisms** are how Denizen parses and manages data related to objects. Here are a few examples of their use in script containers:
```

Comment:
"Write data" and "Change data" mean the same thing.
This entire introduction doesn't really make sense.
A mechanism is a way to set a value on an object, basically a mini-command, that works with the property system ...